vnext/Microsoft.ReactNative/ReactNativeHost.idl

```diff
@@ -20,5 +36,9 @@ namespace Microsoft.ReactNative {
     Windows.Foundation.IAsyncAction LoadInstance();
     Windows.Foundation.IAsyncAction ReloadInstance();
     Windows.Foundation.IAsyncAction UnloadInstance();
+
+    event Windows.Foundation.EventHandler<InstanceCreatedEventArgs> InstanceCreated;
```

Comment:
>InstanceCreated [](start = 68, length = 15)

I was thinking that we will expose them on the ReactInstanceSettings class instead.

Reply:
The difference being that they would be snapshotted on instance creation, right?

That kind of makes sense, but then the user can no longer unregister very easily. - Thoughts?